jdaviz/configs/imviz/helper.py

```diff
@@ -177,14 +187,168 @@ def offset_to(self, dx, dy, skycoord_offset=False):
                     new_sky_cen = sky_cen.spherical_offsets_by(dx, dy)
                 self.center_on(new_sky_cen)
             else:
-                raise AttributeError(f'{image.label} does not have a valid WCS')
+                raise AttributeError(f'{getattr(image, "label", None)} does not have a valid WCS')
         else:
             with delay_callback(viewer.state, 'x_min', 'x_max', 'y_min', 'y_max'):
                 viewer.state.x_min += dx
                 viewer.state.y_min += dy
                 viewer.state.x_max = viewer.state.x_min + width
                 viewer.state.y_max = viewer.state.y_min + height
 
+    @property
+    def marker(self):
```

Comment:
Is this a property because of the astrowidgets idiom where you add markers by clicking? If so, this is rather tricky because I think in this API for `imviz` it makes more sense to have the `marker` property be set at `add_marker` since that's the only time it gets used.

So one option is to remove it entirely as a property, or make it read-only, although that breaks astrowidgets compatibility... See my comment in `add_marker` for a less drastic choice.

Reply:
(resolving this with no action needed because out-of-band @pllim talked me out of this)

Reply:
Yes, we have agreed to pursue this over at astropy/astrowidgets#141 first and open follow-up PR here if necessary.